src/modules/fancyzones/lib/ZoneWindow.cpp

```diff
@@ -621,7 +621,7 @@ void ZoneWindow::OnPaint(HDC hdc) noexcept
     COLORREF hostZoneBorderColor{};
     COLORREF hostZoneHighlightColor{};
     int hostZoneHighlightOpacity{};
-    std::vector<winrt::com_ptr<IZone>> zones{};
+    std::map<size_t, winrt::com_ptr<IZone>> zones{};

```

Comment:
why not use ZonesMap here? nit: `{}` is unnecessary here, since std::map isn't a POD type.